pkg/controller/cloud/pvlcontroller.go

```diff
@@ -196,14 +190,14 @@ func (pvlc *PersistentVolumeLabelController) addLabelsToVolume(vol *v1.Persisten
 	// Only add labels if in the list of initializers
 	if needsInitialization(vol.Initializers, initializerName) {
 		if vol.Spec.AWSElasticBlockStore != nil {
-			labels, err := pvlc.findAWSEBSLabels(vol)
+			labels, err := (pvlc.cloud).(PVLabeler).GetLabelsForVolume(vol)
 			if err != nil {
 				return fmt.Errorf("error querying AWS EBS volume %s: %v", vol.Spec.AWSElasticBlockStore.VolumeID, err)
 			}
 			volumeLabels = labels
 		}
 		if vol.Spec.GCEPersistentDisk != nil {
-			labels, err := pvlc.findGCEPDLabels(vol)
+			labels, err := (pvlc.cloud).(PVLabeler).GetLabelsForVolume(vol)
```

Comment:
please don't do the conversion directly. Check if `pvlc.cloud` can be casted to `PVLabeler` first.
Otherwise all clouds that doesn't implement that iface will panic here

Reply:
Done